### PR TITLE
Normalize Resources api

### PR DESCRIPTION
**Pull request by sjmiles**

### CODE REVIEW

**Review by jinjingforever — APPROVED** [review]

**Review by mariakleiner — APPROVED** [review]
so much better, thank you!
🔥 

PS: please, address comments before submitting.

File: pkg/Library/DevTools/Dom/resource-view.js [hunk 1]
```diff
@@ -14,8 +15,7 @@ export class ResourceView extends Xen.Async {
     return ['version'];
   }
   update(_, state) {
-    state.resources = globalThis.resources || {};
-    state.canvases = entries(state.resources).filter(([name, res]) => res?.localName === 'canvas');
+    state.canvases = entries(Resources.all()).filter(([name, res]) => res?.localName === 'canvas');
```
[the remark this answers is not in the document]
Comment (sjmiles) [reply pkg/Library/DevTools/Dom/resource-view.js:18]: Probably. There are still a few loose ends. This particular method is only in Devtools, so it's low priority.

File: pkg/Library/Firebase/tryst.js [hunk 2]
```diff
@@ -30,16 +26,10 @@ export const meetStrangers = async (nid, meta) => {
...
   return beacons;
-  //
-  //const ids = keys(beacons);
-  // if (ids.length) {
-  //   console.log(`tryst[${nid}]:beacons`, pretty(ids));
-  // }
-  //return ids;
 };
 
 const fetchBeacons = async () => {
```
[the remark this answers is not in the document]
Comment (sjmiles) [reply pkg/Library/Firebase/tryst.js:31]: This is all repaired in follow up PRs.

File: pkg/Library/Firebase/tryst.js [hunk 3]
```diff
@@ -30,16 +26,10 @@ export const meetStrangers = async (nid, meta) => {
...
-  const res = await fetch(`${tryst}/beacons.json`);
+  const res = await fetch(`${tryst}/aeon/beacons.json`);
```
[the remark this answers is not in the document]
Comment (sjmiles) [reply pkg/Library/Firebase/tryst.js:32]: Will be fixed.

**Pull request merged**